release-notes/v1_62.md

```diff
@@ -81,6 +81,24 @@ There is a new setting `html.completion.attributeDefaultValue` that lets you cho
 
 Work continues on the [GitHub Pull Requests and Issues](https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-pull-request-github) extension, which allows you to work on, create, and manage pull requests and issues. Check out the [changelog for the 0.32.0](https://github.com/microsoft/vscode-pull-request-github/blob/main/CHANGELOG.md#0320) release of the extension to see the highlights.
 
+### Jupyter
+
+**Kernels**
+
+The list of kernels displayed in the kernel picker is now grouped to make it easier for one to identify and find kernels belonging to a specific Python envoironment.
+
+![Kernels grouped in kernel picker](images/1_62/kernel_picker.png)
```

Comment:
Does this relative path thing work live? With other images I thought that we had to add the full URL from https://github.com.

Reply:
Hmm, those were all <img> tags in the Readme.md. As long as this link works when hosted live I'm down.

Reply:
@IanMatthewHuff the images should work relative like this I believe. @gregvanl?

Reply:
Yes, others have relative paths

Reply:
Yes, images work with relative paths and the format is correct. 
Doc links need to be absolute for the release note when viewed in-product.